--- a/src/db/schema.rs
+++ b/src/db/schema.rs
@@ -25,7 +25,7 @@
         id -> Int8,
         submission_id -> Int8,
         tag -> Text,
-        requested_as -> Array<Nullable<Text>>,
+        requested_as -> Array<Text>,
         eligible_at -> Nullable<Timestamptz>,
         voided_at -> Nullable<Timestamptz>,
         assigned_runner_id -> Nullable<Int4>,
@@ -70,8 +70,8 @@
     submissions (id) {
         id -> Int8,
         submitted_at -> Timestamptz,
-        requested_tags -> Array<Nullable<Text>>,
+        requested_tags -> Array<Text>,
         origin_id -> Int8,
         report -> Nullable<Json>,
     }
 }
